functions/src/logs.ts

```diff
@@ -0,0 +1,83 @@
+'use strict'
+/*
+ * Copyright 2021 MeiliSearch
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { logger } from 'firebase-functions'
+
+import config from './config'
+
+/**
+ * Initialization logger.
+ */
+export function init() {
+  logger.log('Initializing extension with configuration')
```

Comment:
Is it useful to add that the extension is initialised with configuration or is this maybe redundant as it is always initialised with configuration? 

Also is the naming of the project an extension or a plugin ?

Reply:
It's an extension. I copied the functions present in the [google extensions](https://github.com/firebase/extensions/blob/master/firestore-send-email/functions/src/logs.ts). I thought that if users often use extensions it's better to keep some consistency.